docs/tables/steampipecloud_organization_workspace_member.md

```diff
@@ -0,0 +1,75 @@
+# Table: steampipecloud_organization_workspace_member
+
+Organization Workspace members can collaborate and share connections and dashboards.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  id,
+  org_handle,
+  workspace_handle,
+  user_handle,
+  status,
+  role
+from
+  steampipecloud_organization_workspace_member;
+```
+
+### List invited members
+
+```sql
+select
+  id,
+  org_handle,
+  workspace_handle,
+  user_handle,
+  status,
+  role
+from
+  steampipecloud_organization_workspace_member
+where
+  status = 'invited';
+```
+
+### List owners of an organization workspace
+
+```sql
+select
+  id,
+  org_handle,
+  workspace_handle,
+  user_handle,
+  status,
+  role
+from
+  steampipecloud_organization_workspace_member
+where
+  org_handle = 'testorg'
+and
```

Comment:
This formatting looks a little inconsistent, can you please run your SQL queries in this file through the formatter mentioned in https://steampipe.io/docs/develop/table-docs-standards#style-conventions-1?